<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(215,215,158);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbProgName">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>634</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Personal Mobility</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDispDate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2021/11/09</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbDispTime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>11:12:13 AM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbODD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>ODD</string>
    </property>
    <widget class="QLCDNumber" name="lcdDigitNumber">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>19</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispUnit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>61</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>km/h</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispLabelTop">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>130</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>12.C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispLabelMiddle">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>40km/h</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbStateEmergency">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/warning.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbLightInd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/light.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbLeftSig">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/left.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbBattery">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/battery.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbRightSig">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/right.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbTemp">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/temp.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdGear">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lato Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
       <strikeout>true</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white; color : green;</string>
     </property>
     <property name="digitCount">
      <number>1</number>
     </property>
     <property name="intValue" stdset="0">
      <number>1</number>
     </property>
    </widget>
    <zorder>lbDispUnit</zorder>
    <zorder>lbDispLabelTop</zorder>
    <zorder>lbDispLabelMiddle</zorder>
    <zorder>lbStateEmergency</zorder>
    <zorder>lbLightInd</zorder>
    <zorder>lbLeftSig</zorder>
    <zorder>lbBattery</zorder>
    <zorder>lbRightSig</zorder>
    <zorder>lbTemp</zorder>
    <zorder>lcdGear</zorder>
    <zorder>lcdDigitNumber</zorder>
   </widget>
   <widget class="QPushButton" name="btnMainMenu">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>640</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampEmergency">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/system/warning.png);
background : transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirLeft">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>319</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/system/left.png);
background : transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirRight">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>319</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/system/right.png);
background : transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAccel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/system/up.png);
background : transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrake">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>100</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/images/system/down.png);
background : transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Lamp Control</string>
    </property>
    <widget class="QPushButton" name="btnLampAuto">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampOn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>On</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbLightInd_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/light.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbLightInd_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/system/tail.png);
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampOff">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>99</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampManual">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>99</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbCan">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Off</string>
    </property>
   </widget>
   <zorder>lbProgName</zorder>
   <zorder>lbDispDate</zorder>
   <zorder>lbDispTime</zorder>
   <zorder>btnMainMenu</zorder>
   <zorder>btnLampEmergency</zorder>
   <zorder>btnLampDirLeft</zorder>
   <zorder>btnLampDirRight</zorder>
   <zorder>btnAccel</zorder>
   <zorder>btnBrake</zorder>
   <zorder>btnStart</zorder>
   <zorder>lbStatus</zorder>
   <zorder>lbCan</zorder>
   <zorder>grbODD</zorder>
   <zorder>groupBox</zorder>
   <zorder>btnLampDirOff</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
